lib/tic_tac_toe.rb

```diff
@@ -1,15 +1,52 @@
 require_relative './command_line_in'
 require_relative './prompt'
 require_relative './board_presenter'
+require_relative './board'
+require_relative './input_validator'
 
 class TicTacToe
-    def play
+    @@selected_token
+    @@selected_move
+    
+    attr_accessor :board, :board_presenter, :input_validator
+
+    def initialize
+        @board = Board.new
+        @board_presenter = BoardPresenter.new
+        @input_validator = InputValidator.new
```

Comment:
Should these be initialized here or passed in as arguments?

Reply:
I think the presenter and validator can be passed in as argument, but I do think that I should initialize the board.